profile/README.md

```diff
@@ -0,0 +1,28 @@
+# Executable Books
+
+The Executable Books Project is an international collaboration to build open source tools that facilitate publishing computational narratives using the Jupyter ecosystem.
+
+We are a community who steward:
+
+- [Jupyter Book](https://jupyterbook.org): Build beautiful, publication-quality books and documents from computational material like Jupyter Notebooks.
+- [MyST Markdown](https://myst.tools): An extended Markdown format designed for the future of technical communication and publication.
+
+As well as many, many other tools for parsing, headless execution of Jupyter Notebooks ([caching](https://github.com/executablebooks/jupyter-cache) and in the [browser](https://github.com/executablebooks/thebe)), popular sphinx extensions (e.g. [copy-button](https://github.com/executablebooks/sphinx-copybutton), [sphinx-design](https://github.com/executablebooks/sphinx-design), [sphinx-thebe](https://github.com/executablebooks/sphinx-thebe)), and many [scientific templates](https://github.com/myst-templates) for writing journal articles and presentations.
+
+## Contribute to a Project 👩‍💻
+
+[Ask a question](https://github.com/orgs/executablebooks/discussions) or [show and tell](https://github.com/orgs/executablebooks/discussions/categories/show-and-tell) the community about your project or use of MyST, or come to a [monthly team meeting](https://executablebooks.org/en/latest/meetings/index.html).
+
+All of our projects are open source and welcome and and all contributions -- from documenting problems you run into, adding to documentation, [telling people](https://twitter.com/executablebooks) about your project and our mission, and _of course_ contributing to the code! Please review our [Code of Conduct](../CODE_OF_CONDUCT.md) to learn about some of the conventions we follow.
```

Comment:
We should put the code of conduct in a dedicated header above "Contribute to a project".

e.g.:

```
## Code of Conduct

By participating in any space stewarded by the Executable Books Project, you must follow our [Code of Conduct](link). Please review its content and the processes we have for enforcement. However, note this CoC is aspirational, as the project is transitioning to be more inclusive, and is still developing its culture and CoC procedures. 
```